src/main/java/gregtech/api/capability/INotifiableHandler.java

```diff
@@ -30,4 +33,14 @@ default <T> void addToNotifiedList(MetaTileEntity metaTileEntity, T handler, boo
     void addNotifiableMetaTileEntity(MetaTileEntity metaTileEntity);
 
     void removeNotifiableMetaTileEntity(MetaTileEntity metaTileEntity);
+
+    default int size() {
```

Comment:
Should this be non-default and just have manual implementations for the things that need it?

Reply:
the default method should cover basically all cases, otherwise i'd need to implement it 17 times. overriding the method should only be done for special behavior if required